delfin/tests/unit/drivers/ibm/ibm_ds8k/test_ibm_ds8k.py

```diff
@@ -0,0 +1,313 @@
+# Copyright 2021 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import sys
+from unittest import TestCase, mock
+
+sys.modules['delfin.cryptor'] = mock.Mock()
+
+from delfin import context
+from delfin.drivers.ibm.ds8k.rest_handler import RestHandler
+from delfin.drivers.ibm.ds8k.ds8k import DS8KDriver
+
+
+class Request:
```

Comment:
I can't find where this class is used, maybe it can be deleted, please confirm.

Reply:
done

Reply:
done